README.md

```diff
@@ -48,7 +51,7 @@ This will run `cargo build` as well as any other custom build processes we have
 
 The service can listen to an arbitrary socket, as specified in the help command:
 
-```
+```console
```

Comment:
For the future: Maybe we standardize and do everything as `console` instead of alternating between `sh` and `console`?

Reply:
`console` would be for an optional command prefixed with `$` follow by console output, and `sh` would be for just commands, with no `$`.